client/src/components/template/templateCard.component.js

```diff
@@ -2,11 +2,11 @@ import React, { useEffect } from "react";
 import { Link } from "react-router-dom";
 import M from "materialize-css";
 
-String.prototype.trunc =
-  String.prototype.trunc ||
-  function(n) {
-    return this.length > n ? this.substr(0, n - 1) + "&hellip;" : this;
-  };
+// String.prototype.trunc =
```

Comment:
You should remove commented code when you deliver to master.

Reply:
Noted.